data/test-data/CTA-North-LST-1-Current_test-telescope-model.cfg

```diff
@@ -0,0 +1,107 @@
+%==================================================
```

Comment:
Is this what you referred to as "mock test data"? Just wondering what it is.

Reply:
Not exactly. In this case I am just using an existing config file to create a telescope_model, instead of reading from the DB. The mock data framework provided by pytest is fancier than that. But I don't know much details.